src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -160,9 +164,13 @@ private FeedbackSessionsForm buildBasicForm(List<CourseAttributes> courses, Stri
         List<ElementTag> courseIdOptions = getCourseIdOptions(courses, courseIdForNewSession, instructors, newFeedbackSession);
         boolean isSubmitButtonDisabled = courseIdOptions.isEmpty();
         
-        // adds the placeholder option to courseIdOptions if it is empty
-        if (isSubmitButtonDisabled) {
-            addPlaceholder(courseIdOptions);
+        // Adds a placeholder option to courseIdOptions if it is empty
+        if (courseIdOptions.isEmpty()) {
+            if (courses.isEmpty())
+                addPlaceholder(courseIdOptions, Const.StatusMessages.INSTRUCTOR_NO_ACTIVE_COURSES);
+            else {
+                addPlaceholder(courseIdOptions, Const.StatusMessages.INSTRUCTOR_NO_MODIFY_COURSES_SESSION_PERMISSION);
+            }
```

Comment:
Hmmm. Actually previously I was doing:

```
private void addPlaceholder(List<ElementTag> selectOptions, List<CourseAttributes> courses) {
   ElementTag placeholder;  
   if (selectOptions.isEmpty()) {
      if (courses.isEmpty()) {
         placeholder = createOption(Const.StatusMessages.INSTRUCTOR_NO_MODIFY_COURSES_SESSION_PERMISSION, "", true);
      } else {
         placeholder = createOption(Const.StatusMessages.INSTRUCTOR_NO_ACTIVE_COURSES, "", true);
      }
   } 
   return placeholder;
}
```

But was rejected in another PR. Can't really think of any other way to do it without it looking too weird @kanghj should I use make that method mentioned above?